### DISCUSSION TITLE
choosing right technologies

### DISCUSSION THREAD

**Discussion by amantiwari1** [prisma/prisma#7946] [Q&A] [answered]
hi, I need help with building a matrimonial site, I prefer to choose 

- language - typescript 
- frontend -  nextjs  
- backend - prisma 2  + typeGraphql. 

is it a good choice? any suggestion

**Comment by ryands17** [comment, marked as answer]
@amantiwari1 👋 

Yes this is a good choice. You can go for Next.js and build your backend quickly with Prisma and TypeGraphQL. Have you already setup this stack? Let me know if you have any other queries.

**Reply by amantiwari1** [reply]
I have 1 questions - 
 
- how can I upload files for graphql?

**Reply by ryands17** [reply]
The ideal way to upload files would be from the frontend, to a service like S3 or Cloudinary. After uploading, you can run a GraphQL mutation that stores the file names in the database.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
